tests/index-test.js

```diff
@@ -20,7 +20,7 @@ describe('schenkerian', function () {
       expect(response.description).to.equal(MIXCOM_DESCRIPTION)
       expect(response.image).to.exist
       expect(response.amphtml).to.not.exist
-      expect(response.canonical).to.not.exist
+      expect(response.canonical).to.equal('https://mix.com/')
```

Comment:
why did this change?

Reply:
We added canonical meta tags to all our site pages.